quarkus-integration/quarkus/devui-integration-test/pom.xml

```diff
@@ -18,7 +18,7 @@
     <!-- Code of integration tests should not be a part of test coverage reports. -->
     <sonar.coverage.exclusions>**/*</sonar.coverage.exclusions>
     <!-- Quarkus 2 and 3 use different context roots. -->
-    <dev.ui.root>/q/dev-v1</dev.ui.root>
+    <dev.ui.root>/q/dev-ui</dev.ui.root>
```

Comment:
Since `main` only uses Quarkus 3, I think we can go back to hardcoding this and removing the comment.